lerobot/scripts/control_robot.py

```diff
@@ -491,10 +612,11 @@ def on_press(key):
     )
     if run_compute_stats:
         logging.info("Computing dataset statistics")
-        os.system('say "Computing dataset statistics" &')
+        say("Computing dataset statistics")
         stats = compute_stats(lerobot_dataset)
         lerobot_dataset.stats = stats
     else:
+        stats = {}
```

Comment:
Thanks for fixing this, I hit an uninitialized var error when running
```
python lerobot/scripts/control_robot.py record_dataset \
    --fps 30 \
    --root tmp/data \
    --repo-id $USER/koch_test \
    --num-episodes 1 \
    --run-compute-stats 0
```


Also, there is a typo on the next line. `statistrics` should be `statistics`

Reply:
fixed the typo! thanks